.circleci/config.yml

```diff
@@ -26,6 +30,10 @@ jobs:
 
       - run: make build RELEASE_BRANCH=${CIRCLE_BRANCH}
 
+      - run: sudo npm install -g @lhci/cli@0.8.x
+
+      - run: lhci autorun
```

Comment:
should we add `numberOfRuns` flag to [decrease variability](https://github.com/GoogleChrome/lighthouse-ci/blob/main/docs/configuration.md#numberofruns) ?

Reply:
Address in [4ba95cd](https://github.com/twreporter/twreporter-react/pull/2227/commits/4ba95cd01f58f68bd2c1f0ed16936432427f1d49)